src/common/libjob/Makefile.am

```diff
@@ -56,8 +56,7 @@ test_ldadd = \
 	$(LIBPTHREAD) \
 	$(LIBRT) \
         $(FLUX_SECURITY_LIBS) \
-	$(LIBDL) \
-	$(LIBSODIUM_LIBS)
```

Comment:
libjob/sign_none.c uses libsodium, so this should remain.

Another option would be to convert it to use the CCAN base64 just merged.

Reply:
working on #3787, perhaps that should go in first before this